modules/tinymce/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added a new `tablecellborderwidth` toolbar button and menu item to change table cell border width #TINY-7478
 - Added a new `tablecellborderstyle` toolbar button and menu item to change table cell border style #TINY-7478
 - Added a new `tablecaption` toolbar button and menu item to toggle captions on tables #TINY-7479
+- Added a new `mceTableToggleCaption` command that toggles caption on selected table #TINY-7479
```

Comment:
```suggestion
- Added a new `mceTableToggleCaption` command that toggles captions on selected table #TINY-7479
```
to match the line above

Reply:
```suggestion
- Added a new `mceTableToggleCaption` command that toggles captions on a selected table #TINY-7479
```
Maybe? It just reads a little weirdly otherwise.